src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
-'use strict';
+'use strict'
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
```

Comment:
It looks like both promises should be declared at the beginning, but `promise2` is declared inside the `then` of `promise1`. Please move the declaration of `promise2` to the top, and make sure it's rejected instead of resolved. 😊